js/ads.js

```diff
@@ -0,0 +1,83 @@
+// Подключаем шаблон карточки
+const adsTemplate = document.querySelector('#card').content;
+const newCardTemplate = adsTemplate.querySelector('.popup');
+
+// Подключаем блок с картой для вставки тестовой карточки
+const mapTestCard = document.querySelector('.map__canvas')
+
+// Создаём объект с типами жилья
+const offerType = {
+  flat: Квартира,
+  bungalow: Бунгало,
+  house: Дом,
+  palace: Дворец,
+  hotel: Отель,
+};
+
+// Создаём DOM-элементы, соответствующие объявлениям, и заполняем их данными
+
+// Стрелочная функция возвращает метод map, запущенный на переданном массиве данных.
+const getCardElement = (dataCards) => dataCards.map(({author, offer}) => {
```

Comment:
вот как раз сюда ты должен передать одну дату
```suggestion
const getCardElement = ({author, offer}) => {
```